README.rst

```diff
@@ -57,6 +57,37 @@ can find it at `PULL_REQUEST_TEMPLATE.md <.github/PULL_REQUEST_TEMPLATE.md>`_.
 The issue report template should be automatically applied if you are creating an issue on GitHub as well. Otherwise you
 can find it at `ISSUE_TEMPLATE.md <.github/ISSUE_TEMPLATE.md>`_.
 
+Getting Started
+-----------------
+If you want to run dbt-schema-builder locally, clone this repository on your local system.
+You need a python virtual environment to run this, use the following commands to create and activate python 3.6.12 virtual environment
+::
+    $ pyenv install 3.6.12
+    $ pyenv virtualenv 3.6.12 schemabuilder-venv
+    $ pyenv activate schemabuilder-venv
+    $ cd dbt-schema-builder
+
+Next, you will run following commands
+::
+    $ pip install -U pip-tools
```

Comment:
Users shouldn't need to install pip-tools or do a pip-compile to get started, and doing so might upgrade dependencies to a non-working state. They should just be able to do a make upgrade to get the pinned dependencies that are already checked in.

Reply:
FWIW, this was a required step for both jazib and I.  It was some weird compatibility issue with an old version of pip-tools---the makefile would install the old broken pip-tools which would break any/all subsequent lines in the make recipe.  However, I think codifying this one-time fix is not appropriate, so we should remove these steps like you recommend.

Reply:
I will make the changes in this readme file generic as @bmedx suggested, I will write these troubleshooting and warehouse-transform related steps in internal wiki